bibliopixel/remote/control.py

```diff
@@ -48,8 +58,18 @@ def __init__(self, layout, animations,
             adesc['data']['display'] = adesc['data'].get('display', adesc['name'])
             adesc['name'] = normalize_name(adesc['name'])
             anim['run'] = anim.get('run', {})
+            if auto_demo:
```

Comment:
I can't sub class off RemoteControl for this. That would be silly.
Unless you mean sub-class sequence and then just pass all the animations to it?